src/peilbeheerst_model/Shortest_path/08_shortest_path_Wetterskip.py

```diff
@@ -416,10 +416,17 @@ def connect_linestrings_within_distance(gdf, max_distance=4):
         gdf_cross_single.plot(ax=ax, color="orange", label="crossings")
         plt_paths.plot(ax=ax, color="purple", label="shortest paths")
         ax.legend()
-        plt.savefig(f"./shortest_path/Figures/shortest_path_{waterschap}_RHWS_{index}_new", dpi=300)
-
+        path_figs = os.path.join(
+            os.getenv("RIBASIM_NL_DATA_DIR"),
```

Comment:
I see you use `os.getenv("RIBASIM_NL_DATA_DIR")` several times.

Since #310 it is no longer required to set this as an environment variable. I don't have it set either, so this will fail. Instead you can use `ribasim_nl.settings.ribasim_nl_data_dir` or if you are using CloudStorage, its `data_dir` field.